build/presubmit.yaml

```diff
@@ -28,5 +28,10 @@ steps:
   args: ["build/go_check.sh"]
   id: Go code check
 
+- name: "gcr.io/cloud-foundation-cicd/cft/developer-tools@sha256:b007041726d4f2d7e5270b8fcfc47656850242186825611b33ee97b3867d74ed"
+  entrypoint: "sh"
+  args: ["build/gen_check.sh"]
+  id: Generated files check
+
```

Comment:
we have more tests now and we can add the `waitFor: ["-"]` to each step so they run in parallel

Reply:
I'll open a separate CL.

Reply:
Opened https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/pull/263